prow/cmd/phaino/local.go

```diff
@@ -550,3 +546,25 @@ func (opts *options) convertJob(ctx context.Context, log *logrus.Entry, pj prowa
 	}
 	return fmt.Errorf("grace period expired, aborted: %v", ctx.Err())
 }
+
+func getTimeout(optionsTimeout time.Duration, prowJobTimeout time.Duration) time.Duration {
+	if optionsTimeout != defaultTimeout {
+		return optionsTimeout
+	}
+	if prowJobTimeout > 0 {
+		return prowJobTimeout
+	}
+	return optionsTimeout
```

Comment:
nit: logically this is correct as optionsTimeout is equal to defaultTimeout at this point, but might not be as easy to understand than just returning defaultTimeout

Reply:
Thanks for the catch, I agree that returning `defaultTimeout` will make it more clear.